apps/site/snippets/en/download/pnpm.bash

```diff
@@ -0,0 +1,8 @@
+
+# Enable Corepack:
+corepack enable
```

Comment:
Before landing this, I think you should check with @zkochan to see if this is what pnpm wants Node.js to be recommending. Per https://github.com/orgs/pnpm/discussions/7736#discussioncomment-8673017 I think their preference is probably that we point people toward their standalone install script.

Reply:
Where do you see that recommendation for Windows? When I read the install page, it seems to me like they recommend the install command but then either npm or corepack as alternatives if the command fails. The install page is written fairly naturally; I think we need to ask them directly what they want us to write.

Reply:
![image](https://github.com/user-attachments/assets/e7648c25-ce09-4401-9d13-c4a10b9fe03a)

Reply:
My interpretation of that is that the recommendation is:

1. The PowerShell command; but if that fails,
2. Install via npm; or
3. Install via Corepack

But it's arguably ambiguous so ideally we hear from them directly. If they don't respond before we want to land, and we're going to list instructions, I think our snippet should have the PowerShell command first and then a comment like `# If that fails because of Windows Defender, an alternative is to install via npm:` and then the npm command.

I'd leave Corepack unmentioned since it adds extra complexity (needing to enable it, and possibly also install it if we unbundle it) and they seem to prefer that pnpm not be managed by Corepack.

Reply:
I think it is OK to suggest using corepack for pnpm in all cases as long as you suggest corepack for Yarn as well.

Reply:
Appreciate the update here, @zkochan Ill update the code so that it always shows "corepack enable pnpm".

Reply:
And also, yw! Our goal is to create awareness and fairness to the ecosystem and provide choice to the user. As much as I love `npm`, I don't think it is fair for "npm" to be shown as today as the only package manager.

Reply:
Thanks @zkochan. I think in general we want to list the install steps that the various projects recommend, so if what you see for pnpm isn't exactly what you prefer, please feel free to suggest alternative commands. You shouldn't feel any pressure or obligation to prefer Corepack; npm doesn't, after all. I think for our side we just want a snippet that's concise, that doesn't just more than one recommended set of commands unless there's a strong reason to provide multiple options (like the Windows Defender thing).

Reply:
I couldn't have said better. Thanks, Geoffrey!